### DISCUSSION TITLE
Gantt chart [Jun 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2444] [Discord Threads]
> 🙋‍♂️ **Sidney**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Not in our immediate roadmaps for now  
#  
> 🙋‍♂️ **Sidney**  
#### 😥  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/985828173591765033)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
